Wizard improvements

DavDan1 opened the pull request:
PT: https://www.pivotaltracker.com/story/show/178520033
____
### User Story
```
As a user
In order to enjoy the visit to wizard
We improved it and made it more user friendly
```
___
### Changes proposed
___
- Also add in BOTH buttons
- remove company
- Add phone number
- Loading to submit
- More info about agent
- link to LinkedIn
- Make desktop wizard larger
____
### Movie



https://user-images.githubusercontent.com/57390886/122121300-be816a00-ce2b-11eb-850d-048d223b6efa.mp4

LucasKnudsen committed:
Refactors radio button logic
Adds both buttons to WorkingHours and OfficeType

LucasKnudsen committed:
Integrates phone number and adds loader to submit

LucasKnudsen committed:
Adds increased wizard height on focus

DavDan1 committed:
Adds ne link of linkedin
adds translation to wiard
changes margin for jens inquiry info

DavDan1 committed:
fixes the test bugs

DavDan1 committed:
removes commented code

lukeperrin10 committed:
Merge branch 'development' into wizard_improvements

lukeperrin10 committed:
fixes failing tests on eventRequest

christerforsgren91 commented on src/components/Header.jsx:
```diff
@@ -16,7 +16,7 @@ const LandingHeader = ({ landing }) => {
         <div className='menu-wrapper'>
-          <CustomButton dataCy='phone-number'>
+          <CustomButton dataCy='phone-question'>
             <a href='tel:+46812345678'>08-123 456 78</a>
```
031 insted of 08 :)

DavDan1 replied:
done

christerforsgren91 commented on src/components/Inquiry/Answer.jsx:
```diff
@@ -41,6 +42,7 @@ const Answer = ({
 
   const scroll = () => {
     let container = document.getElementById('wizard-container')
+    // !first &&
```
waz is daz?

DavDan1 replied:
gone

lukeperrin10 committed:
reverts wizard to previous question setup
updates tests accordingly
removes un-needed code

lukeperrin10 committed:
removes linkedin and name information

1CIM commented on src/components/Inquiry/Question.jsx:
```diff
@@ -3,10 +3,12 @@ import agent from '../../assets/Jehn.png'
+        <img src={agent} alt='agent' />
+        <p data-cy={dataCy}>{text}</p>
+      </div>
+    </>
```
are the fragments needed?

1CIM replied:
removed

1CIM commented on src/components/Inquiry/Wizard.jsx:
```diff
@@ -1,41 +1,48 @@
+      id='wizard-container'
+      data-cy='wizard'
+      tabindex='0'>
+      {<InquiryCompanySize />}
```
no needs for curlies right

1CIM replied:
removed

1CIM commented on src/components/Inquiry/Wizard.jsx:
```diff
@@ -1,41 +1,48 @@
       {submitMessage ? (
         <>
           <Question dataCy='on-submit-message' text={submitMessage} />
         </>
```
can ommit these frags aswell right

1CIM replied:
removed

1CIM commented on src/modules/Inquiries.js:
```diff
@@ -2,7 +2,11 @@ import axios from 'axios'
+    setLoading(true)
+    setTimeout(() => {
+      return
+    }, 1000000000)
```
is it long enough? X)

1CIM replied:
set to 500ms

1CIM committed:
remove unnecessary fragments,
change timeout on inquiries to 500ms,
remove underline on footer subheaders,
change phone number to link to same as button say
